documentation/source/install/on_premise_solutions.rst

```diff
@@ -9,44 +9,68 @@ Setting Up OpenEBS with Kubernetes on Local Machine
 ---------------------------------------------------
 The following procedure helps you setup and use OpenEBS on a local machine:
 
-1. Install Vagrant Box
+Install a Vagrant Box either by downloading the vagrant file or by cloning the repository.
 
-   To run the kubernetes cluster on local machine, you need a vagrant box. If you do not have vagrant box follow the steps given `here`_.
+To run the Kubernetes cluster on local machine, you need a vagrant box. If you do not have vagrant box perform the procedure given `here`_.
     .. _here: https://github.com/openebs/openebs/tree/master/k8s/lib/vagrant/test/k8s/1.6#installing-kubernetes-16-and-openebs-clusters-on-ubuntu
 
+Installing a Vagrant Box by Downloading the Vagrantfile
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Create a demo directory for example, k8s-demo using the following command.
+   ::
+      mkdir k8s-demo
+
 2. Download OpenEBS Vagrant file using the following command.
 ::
-
+    cd k8s-demo
     $ wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/vagrant/1.7.5/Vagrantfile
 
 3. Bring up k8s Cluster.
 ::
+   ubuntu@ubuntu:~/k8s-demo$ vagrant up
+
+It will bring up a three node Kubernetes cluster with one master and two nodes. Continue with `step 4`_.
```

Comment:
No common directory for the approaches, will have to add the step of ssh to master seperately

Reply:
Done